KlaviyoSwift.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "KlaviyoSwift"
-  s.version          = "2.5.1"
```

Comment:
Sorry I think I lost the thread here...wasn't master on 3.0.0?

Reply:
Hmm maybe you are merging to a branch or something...

Reply:
yeah since the commit to fix the tests where xcode 14 stopped working was also on major release 2.x.x I'm patching that along with major release 3.x.x.